addons/website_sale_stock_wishlist/data/template_email.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" ?>
+<odoo>
+    <template id="availability_email_body">
+        <div id="body">
+            <p>Dear Customer,</p>
+            <p>The following product is now available.</p>
+            <div style="display: flex; justify-content: center; width: 100%;">
+                <a t-attf-href="#{wishlist.product_id.website_url}">
+                    <img t-attf-src="/web/image/product.product/#{wishlist.product_id.id}/image_1920"/>
```

Comment:
Shoudn't the image source be a full URL to be correctly accessed from the received e-mail ?
(i.e. `https://<whatever.company.com>/...`)
(Maybe not, I do not whether odoo e-mail templates somehow implicitly converts `img`'s `src` to embedded attachments)

This question actually applies to all URL in this template.

Reply:
Actually the `_render_encapsulate` method from `mail.render.mixin` replaces the local links

Reply:
Note that those links most of the time got the ICP as base url, not the website domain.
This will be fixed/improved with https://github.com/odoo/odoo/pull/68201